modules/project-docs/pages/compatibility.adoc

```diff
@@ -8,10 +8,8 @@ Plus notes on Cloud, networks, and AWS Lambda.
 {description}
 
 The 3.2 SDK requires Java 8 or later to be installed, earlier versions will not work.
-_Java 11 is recommended_, which has various enhancements like lambda local variable type inference, profiling tools, and updated security features. 
+_Java 17 is recommended_, which has various enhancements like sealed classes, pattern matching for switch expressions (in preview), and further updates and improvements on core libraries.
```

Comment:
I think here, or later, we need to say this is from 3.2.4 - unless @daschl is confident all 3.2.x will be fine on 17?

Reply:
Arun V mentioned from 3.2.4 in the ticket, so it does make sense to say from SDK 3.2.4 onwards  - would be good to get confirmation from @daschl though 👍 .

Reply:
@RichardSmedley have spoken to Michael, best to say from version `3.2.4` since that's  what has been tested, but in *theory* it should work with all of `3.2.x`.

I'll update this now.